includes/wikia/Wikia.php

```diff
@@ -334,7 +334,6 @@ static public function fixDomainName( $name, $language = false, $type = false )
 		$name = strtolower( $name );
 
 		$parts = explode(".", trim($name));
-		Wikia::log( __METHOD__, "info", "$name $language $type" );
 		if( is_array( $parts ) ) {
 			if( count( $parts ) <= 2 ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)